src/pages/posts/p/[page].tsx

```diff
@@ -0,0 +1,70 @@
+import { GetStaticProps } from "next";
+import siteConfig from "../../../../site.config";
+import Layout from "../../../components/layout/Layout";
+import PageMeta from "../../../components/PageMeta";
+import PostCard from "../../../components/PostCard";
+import Pagination from "../../../components/Pagination";
+import { getPosts } from "@static-fns/blog";
+import { PostData } from "src/PostData";
+
+export async function getStaticPaths() {
+  const posts = getPosts();
+  const paths = new Array(Math.floor(posts.length / siteConfig.settings.postsPerPage)).fill(null);
+
+  return {
+    paths: paths.map((_, page) => ({ params: { page: (page + 2).toString() }})),
+    fallback: false
+  };
+}
+
+export const getStaticProps: GetStaticProps = async ({ params }) => {
+  const posts = getPosts();
+  const page: number = Number(params?.page || "2");
+  const pointer: number = (page - 1) * siteConfig.settings.postsPerPage;
+
+  return {
+    props: {
+      posts: posts.slice(pointer, pointer + siteConfig.settings.postsPerPage),
+      page,
+      total: posts.length
+    }
+  };
+}
+
+type Props = {
+  posts: PostData[];
+  page: number;
+  total: number;
+};
+
+export default function WordsRoute({ posts, page, total }: Props) {
```

Comment:
PageRoute?

Reply:
Yeah, I'm gonna refactor a bunch of these export names along with the relative paths. I think the Posts export is the same.